base/i18n/break_iterator_unittest.cc

```diff
@@ -140,7 +140,9 @@ TEST(BreakIteratorTest, BreakWordThai) {
 // dictionary to detect word boundaries in Thai, Chinese, Japanese, Burmese,
 // and Khmer. Due to the size of such a table, the part for Chinese and
 // Japanese is not shipped on mobile.
-#if !(BUILDFLAG(IS_IOS) || BUILDFLAG(IS_ANDROID))
+// Cobalt does not support Chinese/Japanese word breaking yet. This feature
+// requires a big dictionary(cjdict.txt) to support.
+#if !(defined(OS_IOS) || defined(OS_ANDROID) || defined(STARBOARD))
 
 TEST(BreakIteratorTest, BreakWordChinese) {
```

Comment:
If this compiles fine I think we should just use the DISABLED_ prefix for the test that gtest provides (with the comment still added)